src/main/java/com/google/devtools/build/lib/rules/android/AndroidBinary.java

```diff
@@ -1758,10 +1755,17 @@ private static Artifact createShuffleJarActions(
     return dexedClasspath.build();
   }
 
+  // Adds execution info by propagating tags from the target
+  private static SpawnAction.Builder createSpawnActionBuilder(RuleContext ruleContext) throws InterruptedException {
```

Comment:
it doesn't look like anything here actually throws InterruptedException, can all these throws declarations be removed?